plugin.video.hak5/addon.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+#
+# Imports
+#
+import os
+import sys
+import urlparse
+import xbmc
+import xbmcaddon
+
+reload(sys)
+sys.setdefaultencoding('utf8')
+
+LIB_DIR = xbmc.translatePath(
+    os.path.join(xbmcaddon.Addon(id="plugin.video.hak5").getAddonInfo('path'), 'resources', 'lib'))
```

Comment:
You normally place an empty `__init__.py` inside the folders that contain your python code and then you import them in relation to the directory where you are working on.

An example:

```
-resources
-__init__.py
--lib
-- __init__.py
-- apythonfile.py
addon.py
```

In addon.py:
```python
from resources.lib import apythonfile
```